MANIFEST.in

```diff
@@ -10,6 +10,7 @@ include version_helpers.py
 include cythexts.py
 recursive-include dipy *.pxd
 recursive-include fake_pyrex *
+recursive-include . meson.build
```

Comment:
Since you're switching the build backend from setuptools to meson-python here, MANIFEST.in should not be needed anymore. It's still usable as `python setup.py install` but it's probably not strictly necessary to teach setuptools how to produce a good meson-based sdist?